src/main/scala/com/lightbend/conductr/sbt/BundlePlugin.scala

```diff
@@ -395,93 +424,81 @@ object BundlePlugin extends AutoPlugin {
   }
 
   private def getConfig(config: Configuration, forAllSettings: Boolean): Def.Initialize[Task[String]] = Def.task {
```

Comment:
The changes to `getConfig` are with respect to testing whether a configuration file actually does need to be generated, as it was always being generated before.